app/llm/langchain/iris_langchain_chat_model.py

```
@@ -23,24 +29,51 @@ class IrisLangchainChatModel(BaseChatModel):
     request_handler: RequestHandler
     completion_args: CompletionArguments
     tokens: TokenUsageDTO = None
-    logger = logging.getLogger(__name__)
+    logger: Logger = logging.getLogger(__name__)
 
     def __init__(
         self,
         request_handler: RequestHandler,
         completion_args: Optional[CompletionArguments] = CompletionArguments(stop=None),
```

Ruff

Do not perform function call `CompletionArguments` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)